### ISSUE TITLE
Error on start [AssertionDefect]

### ISSUE THREAD

**Issue by ThomasTJdev** [inv2004/ttop#4]
I'm having some trouble getting it to run:

**nimble**
```nim
[user: /tmp] $ nimble install ttop
[user: /tmp] $ ttop
fatal.nim(54)            sysFatal
Error: unhandled exception: procfs.nim(274, 14) `0 == statvfs(cstring parts[1], stat)`  [AssertionDefect]
```

**wget**
```nim
[user: /tmp] $ wget https://github.com/inv2004/ttop/releases/latest/download/ttop
[user: /tmp] $ chmod +x ttop
[user: /tmp] $ ./ttop
fatal.nim(54)            sysFatal
Error: unhandled exception: procfs.nim(274, 14) `0 == statvfs(cstring parts[1], stat)`  [AssertionDefect]
```

**system**
```bash
[user: /tmp] $ nim -v
Nim Compiler Version 1.6.10 [Linux: amd64]
Compiled at 2022-11-21
Copyright (c) 2006-2021 by Andreas Rumpf

git hash: f1519259f85cbdf2d5ff617c6a5534fcd2ff6942
active boot switches: -d:release

# Linux
# gcc version 12.2.0 (GCC)
```

**Comment by inv2004** [comment]
@ThomasTJdev 
Can you please try to run the last `main` branch version? It will fail, but with `cannot get statvfs of ....` message. To understand which mount point failed

Thank you,

**Comment by ThomasTJdev** [comment]
I can see that it might be some of my podman mount points. Those are saved within /var/lib/containers/storage/overlay.

```bash
[user: /tmp] $ git clone git@github.com:inv2004/ttop.git
Cloning into 'ttop'...
remote: Enumerating objects: 499, done.
remote: Counting objects: 100% (63/63), done.
remote: Compressing objects: 100% (46/46), done.
remote: Total 499 (delta 28), reused 41 (delta 14), pack-reused 436
Receiving objects: 100% (499/499), 72.78 KiB | 374.00 KiB/s, done.
Resolving deltas: 100% (221/221), done.
[user: /tmp] $ cd ttop
[user: /tmp/ttop] - [main] $ nimble install
  Verifying dependencies for ttop@0.3.4
      Info: Dependency on illwill@any version already satisfied
  Verifying dependencies for illwill@0.3.0
      Info: Dependency on zippy@any version already satisfied
  Verifying dependencies for zippy@0.10.4
      Info: Dependency on asciigraph@any version already satisfied
  Verifying dependencies for asciigraph@0.1.2
 Installing ttop@0.3.4
   Building ttop/ttop using c backend
   Success: ttop installed successfully.
[user: /tmp/ttop] - [main] $ ttop
cannot get statvfs of /var/lib/containers/storage/overlay with error -1
procfs.nim(276)          diskInfo
```

**Comment by inv2004** [comment]
@ThomasTJdev 
Just fixed here: https://github.com/inv2004/ttop/commit/9b1c1c3c7e800d6630ac337937574203c52961cf

If we cannot find df on the point - lets ignore it

Could you please show your `/proc/mounts` ? I did not expect the type of mounts have `/dev` prefix

**Comment by ThomasTJdev** [comment]
https://github.com/inv2004/ttop/commit/9b1c1c3c7e800d6630ac337937574203c52961cf fixes it! Thanks :)

I cannot share my full `/proc/mounts`. In my latest comment the error message is for Podman containers overlay storage location. (https://docs.podman.io/en/latest/markdown/podman-mount.1.html#example)
Regarding the `/dev` location, then see snippet below:
```bash
...
/dev/loop7 /var/lib/snapd/snap/snapd/17883 squashfs ro,nodev,relatime,errors=continue 0 0
/dev/loop2 /var/lib/snapd/snap/core18/2667 squashfs ro,nodev,relatime,errors=continue 0 0
/dev/loop0 /var/lib/snapd/snap/bare/5 squashfs ro,nodev,relatime,errors=continue 0 0
...
```

**Comment by inv2004** [comment]
Got it. Thank you

Just released `v0.3.5`
